### PR TITLE
APP-4299 - Changed test spec to match new format from the API

### PR DESCRIPTION
**Pull request by skrrp**

### CODE REVIEW

File: src/rest/methods/agent.test.ts [hunk 1]
```diff
@@ -186,7 +186,7 @@ describe('agentCreatePermissions()', () => {
     expect(agentPermissions).toBeTruthy()
     expect(agentPermissions).toHaveLength(4)
     // expect each role to equal the ones we added
-    agentPermissions.map(permission => {
+    agentPermissions.map((permission) => {
```
Comment (herman-toothrot) [inline src/rest/methods/agent.test.ts:189]: did you change that intentionally? if not I'd revert that change

File: src/rest/methods/agent.test.ts [hunk 2]
```diff
@@ -197,7 +197,7 @@ describe('agentCreatePermissions()', () => {
       )
     })
     // expect appAdmin + pinboardAgent permissions to be timeboxed
-    agentPermissions.map(permission => {
+    agentPermissions.map((permission) => {
```
Comment (herman-toothrot) [inline src/rest/methods/agent.test.ts:200]: did you change that intentionally? if not I'd revert that change

File: src/rest/methods/agent.test.ts (hunk 1 above)
Comment (skrrp) [reply src/rest/methods/agent.test.ts:189]: I didn't touch it and PHPStorm doesn't do any optimisations unless you tell it to.

**Comment by skrrp** [comment]
> mmmhh one needs to be forced

Forcing `php-prod` now. This will be merged once the container is built and the tests pass

**Pull request merged**